tsl/test/sql/gapfill-13.sql

```diff
@@ -0,0 +1,1653 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+\set EXPLAIN 'EXPLAIN (COSTS OFF)'
+
+CREATE TABLE gapfill_plan_test(time timestamptz NOT NULL, value float);
+SELECT table_name FROM create_hypertable('gapfill_plan_test','time',chunk_time_interval=>'4 weeks'::interval);
```

Comment:
Don't we need to test this for all versions? It seems to be only added for PG13.

Reply:
Hmm those files are from different testsuite not sure why they were in this PR too. Removed now.